pkg/util/image_prep_utils.go

```diff
@@ -31,14 +31,10 @@ import (
 	"github.com/golang/glog"
 )
 
-const LOCAL string = "Local Daemon"
-const REMOTE string = "Cloud Registry"
-const TAR string = "Tar"
-
-var sourceToPrepMap = map[string]func(ip ImagePrepper) Prepper{
-	"Local Daemon":   func(ip ImagePrepper) Prepper { return DaemonPrepper{ImagePrepper: ip} },
-	"Cloud Registry": func(ip ImagePrepper) Prepper { return CloudPrepper{ImagePrepper: ip} },
-	"Tar":            func(ip ImagePrepper) Prepper { return TarPrepper{ImagePrepper: ip} },
+var sourceToPrepMap = map[string]func(ip *ImagePrepper) Prepper{
+	"Local Daemon":   func(ip *ImagePrepper) Prepper { return DaemonPrepper{ImagePrepper: ip} },
```

Comment:
Where do these strings come from?  I would get rid of them in favor of some function like 
```
func NewPrepperForSource(source string) Prepper
or
func NewPrepperForSource(source ImageSourceType) Prepper
```

Reply:
yeah this is getting refactored out in an upcoming PR. these are just here for logging at this point